### DISCUSSION TITLE
Adding Apps to PowerBI API

### DISCUSSION THREAD

**Discussion by johandeconingmswa** [migueesc123/PowerBIRESTAPI#160] [Ideas]
Is there any idea whether Apps will be added to the list in the API perhaps?
One thing that's also missing in the APPS as per the Microsoft's site, is that it would not show the Assigned users or groups assigned to view the various Apps.
It would help with security and creating users "same as" down the track for us.

**Comment by migueesc123** [comment]
hey! I don't think that we have any plans at the moment to extend the connector, but we completely accept any contributions, so you're more than welcome to send us a PR for us to review and incorporate in the connector
